CHANGELOG.rst

```diff
@@ -22,7 +22,7 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 * Created the project using QUBE
 * Implement queries for affiliation creation in database
 * Implement use case _List Packages_
-
+* Connect customer search view with db connector
```

Comment:
is this something we would need in the log? I would rather opt for the feature itself like:
add feature to search for a customer or so. But then we would need it for all features/use cases we have.

```suggestion
```

Reply:
I agree with @jenniferboedker